pandas/core/indexes/multi.py

```diff
@@ -1775,22 +1775,44 @@ def droplevel(self, level=0):
 
     def swaplevel(self, i=-2, j=-1):
         """
-        Swap level i with level j. Do not change the ordering of anything
+        Swap level i with level j.
+
+        Calling this method does not change the ordering of anything.
 
         Parameters
         ----------
-        i, j : int, string (can be mixed)
-            Level of index to be swapped. Can pass level name as string.
+        i : int, str
+            First level of index to be swapped. Can pass level name as string.
+            Type of parameters can be mixed.
+        j : int, str
+            Second level of index to be swapped. Can pass level name as string.
+            Type of parameters can be mixed.
```

Comment:
Can you also mention the default values here?

Reply:
Done.